flink-runtime/src/test/java/org/apache/flink/runtime/jobmaster/slotpool/DeclarativeSlotPoolServiceTest.java

```diff
@@ -230,14 +228,13 @@ public void testCreateAllocatedSlotReport() throws Exception {
                     declarativeSlotPoolService.createAllocatedSlotReport(
                             taskManagerLocation2.getResourceID());
 
-            assertThat(
-                    allocatedSlotReport.getAllocatedSlotInfos(),
-                    contains(matchesWithSlotContext(simpleSlotContext2)));
+            assertThat(allocatedSlotReport.getAllocatedSlotInfos())
+                    .is(matching(contains(matchesWithSlotContext(simpleSlotContext2))));
```

Comment:
This matcher is simpler so I prefer to drop it and rewrite the verification to be:
```
assertThat(allocatedSlotReport.getAllocatedSlotInfos())
        .allMatches(
                context ->
                        context.getAllocationId()
                                        .equals(simpleSlotContext2.getAllocationId())
                                && context.getSlotIndex()
                                        == simpleSlotContext2.getPhysicalSlotNumber());
```

Reply:
Fixed